Userland/Libraries/LibCrypto/ASN1/ASN1.cpp

```diff
@@ -92,4 +92,118 @@ String type_name(Type type)
     return "InvalidType";
 }
 
+Optional<Core::DateTime> parse_utc_time(const StringView& time)
```

Comment:
This doesn't look very fault tolerant.

Reply:
Or wait, maybe it does. Never mind :D

Reply:
(it was originally returning a `Result<DateTime, UTCTimeParseFailure>` but honestly...who wants to if the minute is missing, or the hours are missing from the site that gave them a broken certificate?)